### PR TITLE
Implement ^FX and ^LH commands, add CLAUDE.md roadmap

### PR DESCRIPTION
**Pull request by auslavs**
## Summary

First batch of the ZPL command roadmap: implements the two simplest missing commands and records the plan for the rest.

- **`^FX` (Comment)** — adds non-printing comments to a label, rendered as `^FX{text}^FS`. New `Comment` type + `Label.FX` factory.
- **`^LH` (Label Home)** — sets the label home reference point that subsequent `^FO` fields are positioned against, rendered as `^LH{x},{y}`. New `LabelHome` type + `Label.LH` factory.
- **`CLAUDE.md`** — new file documenting the project layout, build/test commands, the pattern for adding a command, and a roadmap for the remaining commands (`^A` font selector, `^FB`, `^BQ`, `^GF`).

Both new commands follow the existing pattern: a domain type in `Types.fs`, a `LabelElement` case, a `Label` factory, a `Render.label` branch, and a test.

## Test plan
- [x] `dotnet build Fabra.sln -c Release` — succeeds, 0 warnings
- [x] `dotnet test Fabra.sln` — 6/6 passing (4 existing + 2 new)
- [x] New tests assert `^FX` renders `^FX{text}^FS` and `^LH` renders `^LH{x},{y}`

https://claude.ai/code/session_01JgobeBrjaun7YtC31Cg9vN

---
_Generated by [Claude Code](https://claude.ai/code/session_01JgobeBrjaun7YtC31Cg9vN)_

### CODE REVIEW

File: Types.fs [hunk 1]
```diff
@@ -180,6 +180,19 @@ type BarcodeFieldDefault =
         let ratio = x.Ratio.ToString(CultureInfo.InvariantCulture)
         $"^BY{x.Width},{ratio},{x.Height}"
 
+/// Comment (^FX)
+type Comment =
+    | Comment of string
+    override x.ToString() =
+        let (Comment str) = x
+        $"^FX{str}^FS"
```
[the remark this answers is not in the document]
Comment (auslavs) [reply Types.fs:188]: Keeping the trailing `^FS`. The ZPL II spec is explicit on this — the `^FX` command's Comments section states: *"Correct usage of the `^FX` command includes following it with the `^FS` command."* So `^FX{comment}^FS` is the spec-recommended idiom, not redundant.

The premature-termination risk is real only when a comment is interleaved *within* a field (between `^FO` and its content), which is misuse. Rather than deviate from the spec, I've documented on `Label.FX` that comments belong between fields, and added a regression test (844ff85). An optional terminator would add API surface for an edge case the spec already advises against.

---
_Generated by [Claude Code](https://claude.ai/code)_

File: Label.fs [hunk 2]
```diff
@@ -179,6 +185,33 @@ type Label =
     { Width = w; Ratio = r; Height = h }
     |> LabelElement.BarcodeFieldDefault
 
+  /// <summary>
+  /// Comment (^FX)
+  ///
+  /// The ^FX command adds a non-printing comment to a label format.
+  /// Any data after ^FX up to the next ^ or ~ command is ignored, so a
+  /// ^FS command should follow it (Fabra appends this automatically).
```
[the remark this answers is not in the document]
Comment (auslavs) [reply Label.fs]: Addressed in 844ff85. `^FS` is kept (spec-recommended), and the `Label.FX` doc comment now states the constraint: place a comment between fields rather than between a `^FO` and its field content, since the trailing `^FS` closes the current field.

---
_Generated by [Claude Code](https://claude.ai/code)_

File: Label.fs [hunk 3]
```diff
@@ -179,6 +185,33 @@ type Label =
...
+  /// </summary>
+  /// <param name="c">Non-printing comment text</param>
+  /// <returns>LabelElement.Comment</returns>
+  static member inline FX c =
+    Comment.Comment c
+    |> LabelElement.Comment
+
```
[the remark this answers is not in the document]
Comment (auslavs) [reply Label.fs:202]: Added in 844ff85 — a regression test renders `Label [ Label.FO ...; Label.FX ...; Label.Text ... ]` and asserts the exact output, locking in that `^FX` emits a standalone `^FX{comment}^FS` line and the field ordering is preserved.

---
_Generated by [Claude Code](https://claude.ai/code)_
